specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -7898,6 +7942,48 @@
           "description": "The time at which the regional restorable database account has been deleted (ISO-8601 format)."
         }
       }
+    },
+    "BackupStorageRedundanciesResult": {
+      "description": "Available backup storage redundancies in selected location.",
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The selected location."
```

Comment:
please be more descriptive

Reply:
Any suggestions for the description? I have mentioned "Available backup storage redundancy types in selected location." just above so I use "The selected location" here